frontend-react/src/components/AppWrapper.tsx

```diff
@@ -32,16 +32,16 @@ export const AppWrapper = ({
     return (
         <Security oktaAuth={oktaAuth} restoreOriginalUri={restoreOriginalUri}>
             <AppInsightsProvider>
-                <SessionProvider oktaHook={oktaHook}>
-                    <QueryClientProvider client={appQueryClient}>
+                <QueryClientProvider client={appQueryClient}>
+                    <SessionProvider oktaHook={oktaHook}>
```

Comment:
This ended up being a moot point since we didn't use a useQuery fetch, but I stand by it as a cleanup measure. Nothing inside QueryClientProvider relies on SessionProvider. With the new arrangement, all the external providers (from libraries) are up top, and ours are nested within.